_gitbook/syntax_and_semantics/literals.md

```diff
@@ -1,3 +1,3 @@
-# Literals
+# 字面常數
```

Comment:
之前上程式語言概論老師都稱之為字面常數

今天才知道也稱為常值


根據"literal 常值"以及"literal 字面常數"搜尋結果比較

稱呼為"常值"者較多


另以語句上也是常值較為精減

選擇翻為常值似乎較佳

________________________________
寄件者: David Kuo <notifications@github.com>
寄件日期: 2016年3月30日 上午 07:04
收件者: crystal-tw/crystal-tw.github.io
副本: Pin-Ju Lin
主旨: Re: [crystal-tw/crystal-tw.github.io] 更新左側已翻譯頁面之標題以及字面常數頁面 (#19)


In _gitbook/syntax_and_semantics/literals.md<https://github.com/crystal-tw/crystal-tw.github.io/pull/19#discussion_r57844453>:

> @@ -1,3 +1,3 @@
> -# Literals
> +# 字面常數


這裡用「常值」會不會比較好？或是你有沒有可以參考的文件，看看大家用什麼翻譯比較多

ref: https://msdn.microsoft.com/zh-tw/library/aa664672(v=vs.71).aspx

—
You are receiving this because you authored the thread.
Reply to this email directly or view it on GitHub<https://github.com/crystal-tw/crystal-tw.github.io/pull/19/files/b896a946de8bd2c3c49ba9f4d23b3780936e75cf#r57844453>